phalcon/cache/backend/redis.zep

```diff
@@ -182,39 +192,26 @@ class Redis extends Backend implements BackendInterface
 	 */
 	public function save(keyName = null, content = null, lifetime = null, boolean stopBuffer = true)
 	{
-		var prefixedKey, lastKey, prefix, frontend, redis, cachedContent, preparedContent,
-			tmp, tt1, success, options, specialKey, isBuffering;
+		var redis, lastKey, frontend, cachedContent, preparedContent, ttl, success, specialKey, isBuffering;
+
+		let redis = this->_getRedis();
 
-		if keyName === null {
+		if !keyName {
```

Comment:
All the use of `keyName` in other methods is `if !keyName`, I think this is inconsistent. In `libmemcached` and `memcache`, `if keyName === null` is used in the `save` method too. Maybe all of them should be `if !keyName`?